Dockerfile

```diff
@@ -6,6 +6,7 @@ COPY handler    handler
 COPY main.go  .
 COPY readconfig.go .
 COPY readconfig_test.go .
+COPY auth.go .
```

Comment:
Please vendor this from faas-provider instead. This is a change I made today to enable this work.

Reply:
This was throwing me off.. looks fine.